core/src/main/java/org/bitcoinj/crypto/BIP38PrivateKey.java

```diff
@@ -152,7 +152,6 @@ private ECKey decryptNoEC(String normalizedPassphrase) {
             byte[] key = Arrays.copyOfRange(derived, 32, 64);
             SecretKeySpec keyspec = new SecretKeySpec(key, "AES");
 
-            DRMWorkaround.maybeDisableExportControls();
             Cipher cipher = Cipher.getInstance("AES/ECB/NoPadding");
```

Comment:
@schildbach I'm guessing this line uses JCE (or at least used to)

Reply:
Answering my own question: Grepping for `getInstance("` yields 8 occurences.